lib/PolarSynchronizer.js

```diff
@@ -163,6 +171,10 @@ PolarSynchronizer.prototype.getLastTimeStamp = function(cb) {
 }
 
 PolarSynchronizer.prototype.setLastTimeStamp = function(lastTimestamp, cb) {
+	if (this.customStartTimeStamp){
+		cb(null);
+		return;
+	}
 	fs.writeFile(this.getLastTimeStampFile(), moment(lastTimestamp).toISOString(), { encoding: 'utf8' }, cb);
```

Comment:
Semantically okay, but please do not use return in if branches. Rather use if-else to represent alternatives.